scripts/delete_db_and_setup.sh

```diff
@@ -1,6 +1,29 @@
-rm backend/run/db.sqlite3
-python3 backend/manage.py migrate
+#!/bin/bash -e
+# To run script in Docker call for example "delete_db_and_setup.sh 'docker-compose exec backend'"
+#MODE={$1:-"None"}
+MODE=$1
+if [[  $MODE != 'DEV' && $MODE != 'PROD' ]]; then
+  echo "Supplied mode is not supported. We only support 'DEV' and 'PROD'"
+  exit 1
+fi
+if [ $MODE == 'DEV' ]; then
+  echo "DO NOT USE THIS SCRIPT IN PRODUCTION"
```

Comment:
```suggestion
  echo "DO NOT USE THE 'DEV' MODE IN PRODUCTION"
```
@bjrne How about this?

Reply:
Sounds funny, but better than everything I've come up with.